core/config/v2/validate.go

```diff
@@ -35,7 +35,6 @@ func validate(s interface{}) (err error) {
 	v := reflect.ValueOf(s)
 	if t.Kind() == reflect.Ptr {
 		if v.IsNil() {
-			//TODO error if required? https://app.shortcut.com/chainlinklabs/story/33618/add-config-validate-command
```

Comment:
I opened a separate story for this and I think it's rather optional.
https://app.shortcut.com/chainlinklabs/story/51095/use-struct-tags-to-validate-config-fields